examples/basic/index.html

```diff
@@ -23,8 +23,9 @@
       Learn more about React at
       <a href="https://facebook.github.io/react" target="_blank">facebook.github.io/react</a>.
     </p>
-    <script src="../../build/react.js"></script>
-    <script src="../../build/react-dom.js"></script>
+
+    <script src="../react-loader.js"></script>
```

Comment:
I've only done it for the basic example, but I could imagine doing this everywhere. We could also add an extra flag to pull in Babel.